Bezier.h

```diff
@@ -1,6 +1,12 @@
 #pragma once
 
 #include "Vector2.h"
+#include <vector>
+
+// Number of curves to use when creating the bezier i.e. smoothness
+#define CURVES 20
+// Width of the track segments
+#define WIDTH 40
```

Comment:
It's best to avoid using `#define` for numerical constants (it's a hangover from C) -- better to declare them as `const` variables instead.